$OpenBSD: patch-js_ui_sessionMode_js,v 1.3 2013/06/10 15:21:06 jasper Exp $

XXX fix this the correct way (if HAVE_NETWORK_MANAGER) and push it upstream

--- js/ui/sessionMode.js.orig	Fri Jun  7 21:45:07 2013
+++ js/ui/sessionMode.js	Mon Jun 10 17:12:35 2013
@@ -102,7 +102,7 @@ const _modes = {
         isLocked: false,
         isPrimary: true,
         unlockDialog: imports.ui.unlockDialog.UnlockDialog,
-        components: ['networkAgent', 'polkitAgent', 'telepathyClient',
+        components: ['polkitAgent', 'telepathyClient',
                      'keyring', 'recorder', 'autorunManager', 'automountManager'],
         panel: {
             left: ['activities', 'appMenu'],
